docs(terminology): Updates n/a and -- descriptions for clarity. (#3783)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/design-guidelines/content/terminology.md

| **U.S.** | As a noun, use *United States* unless there are space constraints. As an adjective, use *U.S.* (for example, *U.S. soldier*). As part of an organization, use *U.S.* <br /><br /> Taken from Red Hat Corporate Style Guide. | 
| **URL** | Stands for *Uniform Resource Locator*. Write *URL* in all caps. |
| **VM** | Refers to a virtual machine. OK to abbreviate as long as you've spelled it out once in the first occurrence and as long as *VM* won't be confused with other terms that share that acronym. <br /><br />Taken from Red Hat Corporate Style Guide. |

> | **N/A** | Refers to *data not applicable*, meaning data is not relevant or does not currently apply to the object.<br /><br />This includes scenarios where there is no matching data in the object's data source. For example, when nothing matches a filter, when the data source is empty, or when the object is not connected to any data source.|

Author: ```suggestion
| **N/A** | Refers to *data not applicable*, meaning there is no relevant, matching, or applicable data.<br /><br />For example, when a filter returns no results, when a data source is empty, or when no data source is connected.|
```

> | **--** | Refers to *data not available*, meaning data could apply to the object, but is currently not accessible and cannot be obtained at the moment.<br /><br />This includes scenarios where the object is connected to a data source, but is not able to fetch data. For example, when there are permission restrictions or errors.|

Author: ```suggestion
| **--** | Refers to *data unavailable*, meaning the data exists and fits the criteria, but is not currently available to display.<br /><br />For example, when a data source is connected, but data can't be fetched due to permission restrictions or errors.|
```
